CMakeLists.txt

```diff
@@ -88,7 +87,13 @@ option(LITECORE_PERF_TESTING_MODE "Build LiteCore with more things public than i
 option(BUILD_ENTERPRISE "Set whether or not to build enterprise edition" OFF)
 option(LITECORE_DISABLE_ICU "Disables ICU linking" OFF)
 option(DISABLE_LTO_BUILD "Disable build with Link-time optimization" OFF)
-option(LITECORE_BUILD_TESTS "Builds C4Tests and CppTests" ON)
+
+if (CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
Note that this is NOT debug by default like you'd think so if no build type is specified it will not enter here.